sklearnex/preview/ensemble/forest.py

```diff
@@ -563,14 +566,20 @@ def _onedal_gpu_supported(self, method_name, *data):
             ready, X, y, sample_weight = self._onedal_ready(*data)
             if not ready:
                 return False
+            elif sp.issparse(X):
+                return False
             elif sp.issparse(y):
                 return False
+            elif sp.issparse(sample_weight):
+                return False
             elif not sample_weight:  # `sample_weight` is not supported.
                 return False
             elif not self.ccp_alpha == 0.0:
                 return False
             elif self.warm_start:
                 return False
+            elif self.oob_score:

```

Comment:
We don't support it for GPU?

Reply:
No, OOB score needed by sklearn are not computed on GPU.